environments/base/manifests/loki-distributor-grpc-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/instance: observatorium-xyz
+    app.kubernetes.io/name: loki
+    app.kubernetes.io/part-of: observatorium
+    app.kubernetes.io/version: master-815c475
+  name: observatorium-xyz-loki-distributor-grpc
+  namespace: observatorium
+spec:
+  clusterIP: None
+  ports:
+  - name: grcp
+    port: 9095
+    targetPort: 9095
+  selector:
+    app.kubernetes.io/component: observatorium-xyz-loki-distributor
```

Comment:
It looks like all of the loki manifests are using the name of the resource as the component label, but this is not the pattern we follow. According to the recommended labels from K8s, this should be the _logical_ purpose of this resource in the stack, i.e. `logs-ingestion`, `load-balancer` etc

Reply:
Good catch!